products/firefox_iOS/README.md

```diff
@@ -0,0 +1,7 @@
+# L10n and Firefox for iOS
+
+Some documentation specific for common tasks regarding Firefox for iOS and l10n:
+* [Localizing Firefox for iOS](localize_iOS.md)
+* [Testing Firefox for iOS](testing_iOS.md)
+* [Adding a locale to builds](adding_iOSlocale.md)
+* [Reviewing new strings for a new release](review_iOSstrings.md)
```

Comment:
Let's keep only the link to existing doc, we can add the others later when the file is available